src/@next/SideSheet/SideSheetStyle.ts

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components';
+import { Neutral } from '../utilities/colors';
+import * as Breakpoints from '../utilities/breakpoints';
+
+export const StyledSideSheetWrapper = styled.div`
+  position: fixed;
+  inset: 0;
+  background-color: rgba(45, 45, 45, 0.5);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  z-index: 999;
+`;
+
+export const StyledSideSheetContainer = styled.div`
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  width: 480px;
+  height: 90vh;
+  background: ${Neutral.B100};
+  box-shadow: 0px 26px 80px rgba(0, 0, 0, 0.2), 0px 0px 1px rgba(0, 0, 0, 0.2);
+  overflow: hidden;
```

Comment:
TODO for me:
- make side sheet shown to the right on large screen
- add animation based on figma